CHANGES.md

```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## 1.10.11 (core) / 0.26.11 (libraries)
+
+### New
+
+- [experimental][dagster-dbt] `DbtProjectComponent` has been reworked, changing both the python api and the yaml schema. `dbt` has been replaced with `project` with a slightly different schema, and `asset_attributes` with `translation`.
```

Comment:
this one should go down to the dg & components section, can drop the `[experimental]` prefix